### DISCUSSION TITLE
异常: too many filtered rows

### DISCUSSION THREAD

**Discussion by 435933750** [apache/doris#18024] [A - General / Q&A]
使用spark connector 往doris里倒入数据。10w/qps, 倒入十分钟左右，就开始报错

be.warning.log->
stream_load.cpp:144] handle streaming load failed, id=6a4a2dfe52826289-668f3290aeff8d99, errmsg=too many filtered rows
除此之外没有任何信息。请问是什么问题？压力太大？有什么方法可以看到更详细的错误信息吗？

be.info.log ->
![image](https://user-images.githubusercontent.com/12441317/226830557-dd4af0e0-8b5c-4e88-8b56-39fd1e91f72b.png)

**Comment by bingquanzhao** [comment]
您好，streaming load 如果出现 too many filtered rows 的问题是由于导入的非法数据大于阈值导致的
<img width="758" alt="image" src="https://user-images.githubusercontent.com/39845445/226845213-73ebb6c0-f392-4416-a76d-610f44edf79d.png">
同时可以查看 https://doris.apache.org/zh-CN/docs/data-operate/import/import-way/stream-load-manual 文档，发现 max_filter_ratio默认值为 0
可以检查一下导入的数据是否存在过多非法脏数据，希望能够帮助到你

**Reply by 435933750** [reply]
我使用的是 spark-doris-connector , 在该组件的源码配置类里 org.apache.doris.spark.cfg.ConfigurationOptions   并未发现可以发配置该参数，这如何配置呢

**Reply by LemonLiTree** [reply]
doris.max.filter.ratio

**Reply by 435933750** [reply]
感谢各位！
